docs/about/team/internships.md

```diff
@@ -35,6 +35,19 @@ sidebar_position: 4
 
 > Notre maître de stage nous a accueillis chaleureusement, et l'ambiance au sein de l'équipe est très agréable. Puisque le stage se déroule dans un environnement bilingue, cela constitue en une excellente opportunité de perfectionner son anglais.
 
+### [Kabir, Ibrahim](www.linkedin.com/in/ibrahimkabir) - Backend Hiver 2023 
+
+**Salut tout le monde !** 👋 
```

Comment:
les deux fichiers ont une signification autre que version anglaise et française du même texte.

internships.md est un outil de marketing

the-software-development-team est l'index des membres de l'équipe

Reply:
Veux-tu retirer les anciens membres du document the-software-development-team puisqu'il est l'index des membres de l'équipe ?

Reply:
en fait, il ne fallait pas modifier internships.md puisque c'est des testimonials qui peuvent rester là.

c'est plutôt dans la liste d'équipe qu'il fallait déplacer ceux qui ne sont plus actifs dans l'équipe dans une section à part.

Reply:
Documentation de First Pull Request mis à jour avec cette information.
Section des anciens membres déplacé dans `the-software-development-team`